sphinx/source/start_network.rst

```diff
@@ -1,135 +1,102 @@
 
-.. code-block:: bash
-
-    $ genesisgenerator tx --members=member*cert.pem --users=user*cert.pem --nodes=nodes.json --gov-script=src/runtime_config/gov.lua --tx0=tx0 --start-json=startNetwork.json
-
-This command also generates the ``startNetwork.json`` RPC file required to start up the network.
+When CCF is used to run a custom Lua application, the starting node should also be started with the ``--app-script=/path/to/lua/application_script`` (see the `samples folder <https://github.com/microsoft/CCF/tree/master/samples/apps>`_ for example of Lua applications).
 
-Starting up the network
-~~~~~~~~~~~~~~~~~~~~~~~
-
-Once the initial nodes are running and the initial state of the network is ready to deploy, the network can be started by one of the members:
-
-.. code-block:: bash
-
-    $ client --server-address=node0_ip:node0_rpcport startnetwork --ca=node0_cert_file --req=@startNetwork.json
-
-When executing the ``startNetwork.json`` RPC request, the target node deserialises the genesis transaction and immediately becomes the consensus primary of the new single-node network. Business transactions can then be issued by users and will commit immediately.
-
-Adding nodes to the network
+Joining an existing network
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Once a network has been started on one node, assuming that this node remains primary of the consensus network, join network RPC files can be generated for all others nodes defined in the initial state of the network (``nodes.json``):
+To join an existing network, other nodes should be started with the ``join`` option:
 
 .. code-block:: bash
 
-    $ genesisgenerator joinrpc --network-cert=networkcert.pem --target-address=node0_ip:node0_rpcport --join-json=joinNetwork.json
+     $ cchost --enclave-file=/path/to/enclave_library --enclave-type=debug
+    --node-address=node_ip:node_port --rpc-address=rpc_ip:rpc_port
+    --public-rpc-address=public_rpc_ip:public_rpc_port --ledger-file=/path/to/ledger
+    --node-cert-file=/path/to/node_certificate --quote-file=/path/to/quote
+    join --network-cert-file=/path/to/existing/network_certificate --target-rpc-address=target_rpc_ip:target_rpc_port
 
-Once done, each additional node (here, node 1) can join the existing network by running the following command:
+The node takes the certificate of the existing network to join via ``--network-cert-file`` and initiates an enclave-to-enclave TLS connection to an existing node of the network as specified by ``--target-rpc-address``. Once the join protocol [#remote_attestation]_ completes, the joining node becomes part of the network as a backup (see :ref:`Ledger replication` for more details on consensus protocols).
 
-.. code-block:: bash
+.. note:: When starting up the network or when a node joins an existing network, the network secrets required to decrypt the ledger are sealed to disc so that the network can later be recovered. See :ref:`Catastrophic Recovery` for more details on how to recover a crashed network.
```

Comment:
I wasn't sure about disc/disk in the abstract, but here I definitely think it should be disk. [This](https://writingexplained.org/disc-or-disk-difference) says even in English (Traditional), the k is used for computer terminology.

Reply:
"sealed and written to a file" is a good way to avoid any trouble.